src/components/CourseTable/index.jsx

```diff
@@ -32,7 +32,7 @@ const CourseTable = () => {
 
   const formatCourseData = courses => courses.map(course => ({
     ...course,
-    title: (<Link to={`/courses/${course.uuid}/edit/`}>{course.title}</Link>),
+    title: (<Link to={`/courses/${course.uuid}`}>{course.title}</Link>),
```

Comment:
Thoughts on trailing slash or not?

Reply:
I'm not a huge fan? But could be persuaded otherwise.

But if we add them, I'd like to be consistent and add them everywhere. So `/instructors/new/`, etc. We were a mishmash before.

Reply:
Definitely agree with consistency, but I personally don't have strong feelings one way or the other.

I believe without the `exact` it can match regardless of the trailing slash so is that potentially the preferred way and internally we remove the slashes everywhere? I remember being annoyed on Discovery when I tried to go to a url that 404'd or something because I forgot the trailing slash and it took me a few minutes to figure out what was happening

Reply:
Even with `exact`, it will match with or without trailing slash.

Reply:
I believe `exact` just prevents <uuid>/blarg from matching.

Reply:
That feels like common sense. Weird that you have to specify that and the default is false.

All good then! And we just need to make sure we start using no trailing slash everywhere

Reply:
Agreed. I don't get it.